### ISSUE TITLE
Tagprefix always white

### ISSUE THREAD

**Issue by Heliosard** [NEZNAMY/TAB#142]
**Server version**
Paperclip: git-Paper-201 (MC: 1.15.2)
Travertine: Build#135

**Plugin version**
2.7.4 (premium)

**Describe the bug**
I use the "bukkit-bridge" mode to share the tablist between several servers of the same group (like hubs) and simplify the management of the scoreboards.
The tagprefix only appears after a `/btab reload` for players already connected to the server where the command was entered. Each new player who connects will find a white tagprefix.

```
Groups:
  _OTHER_:
    tagprefix: "%vault_rank% "
```

I tested by changing the `disable_entity_metadata_rewrite` value in config waterfall.yml but no change.

Here is a screen that illustrates the problem.

![image](https://user-images.githubusercontent.com/31409151/80086081-7b927200-8559-11ea-8b46-daa1ab9e6212.png)

Thanks for your help,

**Comment by NEZNAMY** [comment]
Looks like you don't have bukkit bridge enabled as double nametag can only happen with unlimited nametag mode which isn't supported on bungee.

**Comment by Heliosard** [comment]
In my issue, I wrote this, after checking our configurations.

> I use the "bukkit-bridge" mode to share the tablist between several servers of the same group (like hubs) and simplify the management of the scoreboards.

I'm not talking about the "nlimited nametag mode" function, I'm just talking about the "tagprefix" function that doesn't work in bungee. After each server change after a `/btab reload`, the tagprefix disappears.

**Comment by Tanguygab** [comment]
you maybe have another plugin overriding prefixes, can you send your plugin list (on spigot and bungeecord)

**Comment by NEZNAMY** [comment]
It's the same bug on waterfall which this person reports over and over.

**Comment by Heliosard** [comment]
@Tanguygab Hi,
Thank you for helping us and telling us where to look.

Following your advice, we conducted several tests. We have removed the bungeecord plugins one by one, without finding any problem. Then we did the same thing on the bukkit servers. It seems that the problem (tab order + color tag) appears when ProtocolSupport is installed. When we were using the bukkit version of the TAB plugin, ProtocolSupport did not cause any problem.

I don't know if the problem comes from TAB or ProtocolSupport.

Thanks,
